runtime/src/main/java/org/corfudb/runtime/object/CorfuCompileProxy.java

```diff
@@ -133,7 +139,7 @@
      */
     @Deprecated // TODO: Add replacement method that conforms to style
     @SuppressWarnings("checkstyle:abbreviation") // Due to deprecation
-    public CorfuCompileProxy(CorfuRuntime rt, UUID streamID, Class<T> type, Object[] args,
+    public CorfuCompileProxy(CorfuRuntime rt, UUID streamID, String streamName, Class<T> type, Object[] args,
```

Comment:
what the difference between `streamID` and `streamName` ?
Isn't it a string representation of UUID by the chance?

Reply:
Ok, `streamName` not equals to `streamID`